.github/workflows/integrations.yml

```diff
@@ -250,7 +257,7 @@ jobs:
 
           cmake -S . -B build -DCMAKE_BUILD_TYPE=RelWithDebInfo
           -DOPENASSETIO_ENABLE_TESTS=ON
-          -DOPENASSETIO_GLIBCXX_USE_CXX11_ABI=OFF
+          -DOPENASSETIO_GLIBCXX_USE_CXX11_ABI=${{ matrix.config.python_version == '3.9' && 'OFF' || 'ON' }}
```

Comment:
I don't understand this syntax.

How does
`matrix.config.python_version == '3.9' && 'OFF'` evaluate?

Reply:
It's actually [documented by GitHub as a "ternary operator"](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/expressions#example)

This is fairly idiomatic ternary syntax in Javascript, in my experience, and also has an equivalent in Python
 ```
>>> True and "its truthy" or "its falsey"
'its truthy'
>>> False and "its truthy" or "its falsey"
'its falsey'
```

Reply:
Bit bashy too then. Thanks for the link.

Logically, `<something> && OFF` just feels wrong :P